e2e/kube.go

```diff
@@ -123,7 +123,9 @@ func (k *Kubeclient) WaitUntilPodRunning(ctx context.Context, t *testing.T, name
 			}
 		case event := <-watcher.ResultChan():
 			if event.Type != "ADDED" && event.Type != "MODIFIED" {
-				t.Logf("skipping event %s", event.Type)
+				if event.Type != "" {
```

Comment:
nit: should we have the nil check go first? (put it before line 125?)

Reply:
`event` is of type `watch.Watch` which is a struct, so there isn't any risk of a nil ptr here

Reply:
I didn't make it clear. I mean should we have `event.Type != ""` check go first? before checking "ADDED" and "MODIFIED", though it doesn't change anything in this case.

Reply:
will keep as-is for now, this was mainly to cleanup logs